### ISSUE TITLE
Login user into kde wayland

### ISSUE THREAD

**Issue by imanbarati** [archlinux/archinstall#1742]
I write the right password keep returning to login 
![IMG_20230414_195108_461](https://user-images.githubusercontent.com/40170427/232085953-f360d96e-657b-4ea8-a9c5-f082fbc76a07.jpg)
I added user after installation.

**Comment by julianPescobar** [comment]
Same thing happens to me. No matter what DE I choose.

**Comment by Torxed** [comment]
What's the logs say? Try logging in to a different tty (Ctrl+Alt+F2 or F3, whatever tty is free and available). Try:
`dmesg -w` and `journalctl -f` and see what the logins are complaining about.

**Comment by imanbarati** [comment]
Admin has insufficient privileges can t create DIR /home

**Comment by imanbarati** [comment]
Thanks i ran Useradd -m sa

**Comment by svartkanin** [comment]
I wasn't able to reproduce this, I'll close it for now. If it happens again we can have a look

**Comment by Jan-Bober** [comment]
I encountered same issue. When I install with archinstall Hyprland and session is  Hyprland(uwsm-managed) after login it takes me back to sddm. When session is Hyprland it logs in but SUPER+Q doesn't open kitty(although archinstall says that it isntalls kitty alongside Hyprland), and theres no default Hyprland wallpaper. I also tried with gnome and gdm and problem doesn't exist with this configuration.
